CHANGELOG.md

```diff
@@ -44,6 +47,12 @@
 * feature: validate `<port>.service.kuma.io/protocol` annotations on K8S Service objects
   [#611](https://github.com/Kong/kuma/pull/611)
 
+Breaking changes:
+* ⚠️ before the change TrafficPermission worked in cumulative way, which means that all policies that matched a connection were applied.
+  We changed TrafficPermission to work like every other policy so only "most matching" policy is chosen.
```

Comment:
In the docs we use `"the most specific" matching policy` instead of `"most matching" policy`. I found "the most specific" a bit clearer and descriptive, what do you think?

Reply:
agree, it's cleaner. Let me update the changelog